src/structs/Layer.jl

```diff
@@ -0,0 +1,140 @@
+"""
+    Layer
+
+Defines a new layer withing the video.
+
+# Fields
+- `frames::Frames`: A range of frames for which the `Layer` exists
+- `width::Int`: Width of the layer
+- `height::Int`: hegiht of the layer
+- `position::Point`: initial positon of the center of the layer on the main canvas
+- `children::Vector{AbstractObject}`: Objects defined under the layer
```

Comment:
Can we change "children" to something like "layer_objects"? I want to migrate away from the parent-child-orphan analogies in programming for Javis.